superset-frontend/src/dashboard/components/gridComponents/Tabs.jsx

```diff
@@ -260,6 +263,14 @@ class Tabs extends React.PureComponent {
                       onDeleteTab={this.handleDeleteTab}
                     />
                   }
+                  onExiting={() => {
+                    // Exiting previous tab, animating start
+                    this.props.setIsAnimating(true, tabId);
+                  }}
+                  onEntering={() => {
+                    // Entering current tab, DOM is visible and has dimension
+                    this.props.setIsAnimating(false, tabId);
+                  }}
```

Comment:
Use `onEntering` instead of `onEntered` because the contents for the new tab are already mounted to DOM upon entering and have values for dimension properties.

Reply:
This may be potentially a source of confusion because the animation didn't completely stop when `isAnimating = true`, but I'm struggling to find more appropriate + succinct variable names.

Reply:
maybe they could be `onMount` and `onUnmount`? Or `afterMount` and `beforeUnmount`?

Reply:
Renamed to `isParentMounted` for charts and `isTabMounted` for dashboards.